env0/resource_user_project_assignment.go

```diff
@@ -75,6 +88,13 @@ func resourceUserProjectAssignmentRead(ctx context.Context, d *schema.ResourceDa
 			if err := writeResourceData(&assignment, d); err != nil {
 				return diag.Errorf("schema resource data serialization failed: %v", err)
 			}
+
+			if client.IsBuiltinProjectRole(assignment.Role) {
```

Comment:
same variable in the response. However, in the schema, it goes to a different field.